package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "enigma-bbs",
-  "version": "0.0.12-beta",
+  "version": "0.0.14-beta",
```

Comment:
This line is the enigma version. Though I think we should probably bump to `0.0.13` for this change.

Reply:
Sorry, I was being silly, skipping .13 as an "unlucky number" lol. Changed it back to the actually sane value 0.0.13-beta now.